tests/shunit/twlight_i18n_lint_test.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+shopt -s expand_aliases
+
+testBadPyNewlines() {
+    start=1
+    end=6
```

Comment:
Since the end variable is fixed, I think we should add a comment to mention that if you add a new test case, you need to change this variable so that it is run in the script.

Reply:
I rewrote this to use a pattern instead of a fixed length loop.